samtranslator/model/apigatewayv2.py

```diff
@@ -174,6 +178,14 @@ def generate_openapi(self):
         """
         authorizer_type = self._get_auth_type()
 
+        if authorizer_type == "AWS_IAM":
+            openapi = {
```

Comment:
We should maybe return early here and in the `JWT` case.
So that we don't have to scan the rest of the function to see what we're doing with `openapi`.

Reply:
Yea. I didn't do it that way because we didn't do it that way for any of the other authorizers. I would prefer to make this change, if desired, in another smaller PR.

Reply:
I'm open to that. As a Go developer I prefer early returns but if elif is more python-style I'm open to it. Can we _not_ do it as part of this PR?

Reply:
I prefer moving the openapi object creation to separate private functions, but it is Ok for now.